crates/oxc_transformer/src/es2018/mod.rs

```diff
@@ -0,0 +1,40 @@
+mod object_rest;
+mod object_rest_spread;
+mod object_spread;
+mod options;
+
+pub use object_rest_spread::{ObjectRestSpread, ObjectRestSpreadOptions};
+pub use options::ES2018Options;
+use oxc_ast::ast::*;
+use oxc_traverse::{Traverse, TraverseCtx};
+use std::rc::Rc;
+
+use crate::context::Ctx;
+
+#[allow(dead_code)]
+pub struct ES2018<'a> {
+    ctx: Ctx<'a>,
+    options: ES2018Options,
+
+    // Plugins
+    object_rest_spread: ObjectRestSpread<'a>,
```

Comment:
Should be `Option<ObjectRestSpread<'a>>`, we need this to determine if we should transform